### PR TITLE
Make sure help is shown/unshown correctly for both markdown-generated…

### PR DESCRIPTION
**Pull request by JorisGoosen**
…/helpfile analyses

- fixes https://github.com/jasp-stats/jasp-test-release/issues/1501

### CODE REVIEW

**Comment by boutinb** [comment]
Hou have added the jaspRegression & jaspSummayStatistics modules in your commit.

File: Desktop/utilities/helpmodel.cpp [hunk 1]
```diff
@@ -134,36 +133,36 @@ void HelpModel::showOrTogglePage(QString pagePath)
...
-		return;
-	}
-	
 	QString renderFunc, 
 			contentMD,
 			pagePath = convertPagePathToLower(helpPage == "" ? analysis->helpFile() : analysis->fullHelpPath(helpPage));
 	
-	
-	if(analysis == _analysis && pagePath == _pagePath && _visible)
+	if(!analysis || (analysis == _analysis && pagePath == _pagePath && _visible))
```
Comment (boutinb) [inline Desktop/utilities/helpmodel.cpp]: You test here whether the analysis pointer is not null, but you use this analysis pointer in the line before...

Comment (boutinb) [reply Desktop/utilities/helpmodel.cpp]: pagePath should be set in the else expression

**Review by boutinb — CHANGES_REQUESTED** [review]

File: Desktop/utilities/helpmodel.cpp (hunk 1 above)
Comment (JorisGoosen) [reply Desktop/utilities/helpmodel.cpp]: Oops

**Comment by JorisGoosen** [comment]
> Hou have added the jaspRegression & jaspSummayStatistics modules in your commit.

This was sort of on purpose aas they were part of the fix but are now in development anyhow.

File: Desktop/utilities/helpmodel.cpp (hunk 1 above)
Comment (JorisGoosen) [reply Desktop/utilities/helpmodel.cpp]: I just wanted to remove the duplication...

**Comment by JorisGoosen** [comment]
It should be good now

**Comment by boutinb** [comment]
If I test it from your branch, I still get same problem as EJ: If I click on the help of the Bayesian Log-Linear Regression, I first get a (really short) help, the secind time I don't get anything. If I then go to another analysis, the first time I click on the icon I get the help, I close the help pop-up, then I click again on the help icon: the pop-up does not show up. If I click again, it does show up.

**Comment by JorisGoosen** [comment]
> get a (really short) help

Did you do `git submodule update` ?

Also, I just tested this code here and it works. I can only suspect you either got the wrong branch/didnt-build properly.

Maybe you could share the output of `git log` and `git status` here?

Edit: I've doublechecked it and it does in fact fix the problem for me...

**Review by boutinb — APPROVED** [review]

**Pull request merged**